.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "arrowParens": "always",
+  "trailingComma": "es5",
+  "printWidth": 100
+}
```

Comment:
I'd go with this config, as I use it everywhere else:

```json
{
  "bracketSpacing": false,
  "singleQuote": true
}
```

If you don't have time, I can do it. Also, add the trailing newline, please.

Reply:
I don't mind what formatting you like to use here. You can format them any way you like (I won't have time for that). Anyway, these declarations should be contributed to the official flow-typed repo eventually. Preparing the final version for contribution there I think the best bet would be to format the source code by Prettier with no explicit configuration and going with its defaults ... However, flow-typed repo doesn't have any set code style as far as I know (the declarations in a readable style is what matters)

Reply:
So I'll change and reformat it then. Thanks!